components/common/WalletModal.vue

```diff
@@ -144,6 +145,18 @@ export default class WalletModal extends Vue {
     } else {
       // web3 wallet connect logic here & show accountSelect, async or not?
       // wallet.enable()
+
+      // init account
+      wallet
+        .getAccounts()
+        .then((data) => {
+          this.walletAccounts = data ?? []
+        })
+        .catch((e) => {
+          console.error('init account error', e)
+        })
```

Comment:
same thing is done in `wallet.subscribeAccounts`

Reply:
However, in subwallet, `wallet.subscribeAccounts` will not do the same thing. Therefore the address list could not  be shown.